packages/widgetbook_exception/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: widgetbook_exception
+description: A dart exception handler for Widgetbook
```

Comment:
It's not really an exception handler. it's just a model for a general Exception related to a Widgetbook product.